src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell.binding
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
+  val psTaskWdl = """
+    |task ps {
+    |  command {
+    |    ps
+    |  }
+    |  output {
+    |    File procs = stdout()
+    |  }
+    |}""".stripMargin
+
+  val cgrepTaskWdl = """
+    |task cgrep {
+    |  command {
+    |    grep '${pattern}' ${File in_file} | wc -l
+    |  }
+    |  output {
+    |    Int count = read_int(stdout())
+    |  }
+    |}""".stripMargin
+
+  val wcTaskWdl = """
+    |task wc {
+    |  command {
+    |    wc -l ${File in_file}
```

Comment:
The current `read_int` implementation in `LocalEngineFunctions` would fail to execute this code as written since it wants the entire whitespace-trimmed contents of the file to be parseable as an int.  We should resolve that inconsistency one way or the other.